guides/vim-cheatsheet.rst

```diff
@@ -0,0 +1,444 @@
+Vim Commands Cheat Sheet
```

Comment:
I read a good one today: "I've been using vim since 1996, but only because I can't figure out how to exit."